core/src/main/resources/templates/core/management_general_statistics_for_users.html

```diff
@@ -0,0 +1,125 @@
+    <script src="https://code.jquery.com/jquery-3.5.1.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js"></script>
+    <script src="https://code.highcharts.com/highcharts.js"></script>
+    <script src="https://code.highcharts.com/modules/exporting.js"></script>
+    <script th:src="@{/management/user/graphs.js}"></script>
+</head>
+
+<body>
+<div id="header" th:insert="core/header"></div>
+<div class="main-content">
+    <div class="container">
+        <h2 align="center">Users city Graph</h2>
+
+        <div id="usersCity" style="width: 550px; height: 400px; margin: 0 auto"></div>
+        <hr>
+        <div id="test" style="width: 550px; height: 400px; margin: 0 auto"></div>
+    </div>
+    <script th:inline="javascript" >
+        $(function(){
+            Highcharts.chart('usersCity', {
+                chart: {
+                    type: 'column'
+                },
+                title: {
+                    text: ''
+                },
+                credits: {
+                    enabled: false
+                },
+                xAxis: {
+                    categories: [[${usersByCities.keySet()}]],
+                    crosshair: true
+                },
+                yAxis: {
+                    title: {
+                        text: ''
+                    }
+                },
+                tooltip: {
+                    headerFormat: '<span style="font-size:10px">{point.key}</span><table>',
+                    pointFormat: '<tr><td style="color:{series.color};padding:0">{series.name}: </td>' +
+                        '<td style="padding:0"><b>{point.y}</b></td></tr>',
+                    footerFormat: '</table>',
+                    shared: true,
+                    useHTML: true
+                },
+                plotOptions: {
+                    column: {
+                        pointPadding: 0.2,
+                        borderWidth: 0
+                    }
+                },
+                series: [{
+                    name: 'City',
+                    data: [[${usersByCities.values()}]]
+                }]
+            });
+        });
+    </script>
+    <script th:inline="javascript">
+        Highcharts.setOptions({
+            colors: ['#64E572']
+        });
+        Highcharts.chart('test', {
+            chart: {
+                type: 'line'
+            },
+            title: {
+                text: 'Monthly Average Users Registration'
+            },
+            credits: {
+                enabled: false
+            },
+            subtitle: {
+                text: [[${#dates.year(#dates.createNow())}]]
+            },
+            xAxis: {
+                categories: ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec']
```

Comment:
Should this be localized using one more `Mode Attribute`?

Reply:
Done.